src/main/java/seedu/address/model/person/Phone.java

```diff
@@ -11,8 +11,15 @@ public class Phone {
 
 
     public static final String MESSAGE_CONSTRAINTS =
-            "Phone numbers should only contain numbers, and it should be at least 3 digits long";
-    public static final String VALIDATION_REGEX = "\\d{3,}";
+            "Phone numbers should only contain numbers, and it should be at least 3 or 8 digits long "
+                    + "and start with 6, 8 or 9";
+
+    /**
+     * Phone number regex matches the following conditions:
+     * 1. Has exactly 3 OR 8 digits exactly
+     * 2. Starts with 6, 8 or 9. We ignore numbers starting with 3 since they're not nubmers that patients would have
+     */
+    public static final String VALIDATION_REGEX = "\\b[689]\\d{2}(\\d{5})?\\b";
```

Comment:
I think we should simplify the regex to only take on the case of `\b[689]\d{5}\b` which is 6/8/9 + 7 more digits, otherwise funny numbers like 666, 888 (and 999) will be valid.

Reply:
Hmm, this seems a bit tough. To what extent are we going to validate against fake numbers? For example, 66666666 is also a probably fake number. How do we validate against such numbers and the numbers you mentioned as well?

Reply:
Okay I will approve this PR, but just take note of this problem please - especially the 999 one.